src/Components/Notifications/PartnerOfferArtwork.tsx

```diff
@@ -130,15 +148,35 @@ export const PartnerOfferArtwork: FC<PartnerOfferArtworkProps> = ({
           </Text>
         </Box>
       )}
-      <Box mb={4} width="100%" maxWidth={CARD_MAX_WIDTH}>
+      <Box
+        mb={4}
+        display="flex"
+        justifyContent="space-between"
+        width="100%"
+        maxWidth={CARD_MAX_WIDTH}
+      >
         <Button
           // @ts-ignore
           as={RouterLink}
           to={href}
           data-testid="partner-offer-artwork-button"
+          mr={partnerOfferVisibilityEnabled && fullyAvailable ? 2 : 0}
+          flex={partnerOfferVisibilityEnabled && fullyAvailable ? 1 : [1, 0.5]}
         >
           {buttonText}
         </Button>
+        {partnerOfferVisibilityEnabled && fullyAvailable && (
+          <Button
+            // @ts-ignore
+            as={RouterLink}
+            to={artworkListingHref}
+            data-testid="partner-offer-view-artwork-button"
+            flex={1}
+            variant={"secondaryBlack"}
+          >
+            {"View Work"}
```

Comment:
According to [the design](https://www.figma.com/file/RHkhhJJwixhFPltqhH9ZkW/v1.5-Saves-Partner-Offers?type=design&node-id=2261-24754&mode=design&t=lGsYwiNY83j47vGL-4), this should be `View Artwork`. 

```suggestion
            {"View Artwork"}
```

Reply:
the new designs are [here](https://www.figma.com/file/RHkhhJJwixhFPltqhH9ZkW/v1.5-Saves-Partner-Offers?type=design&mode=design&t=r6goUJCMzvewwlDV-0#757831092)

Reply:
Good to know! I thought the design for the note only got updated.